copilot/post-award-data-store/addons/post-award-data-store-cluster.yml

```diff
@@ -16,11 +16,11 @@ Parameters:
     # Cannot have special characters
     # Naming constraints: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Limits.html#RDS_Limits.Constraints
 Mappings:
-  postawarddatastoreclusterEnvScalingConfigurationMap: 
+  postawarddatastoreclusterEnvScalingConfigurationMap:
```

Comment:
should we rebase / squash these changes into the initial commit?

Reply:
Good point, yes will squash and merge